### PR TITLE
Missing cd command in git clean up documentation

### PR DESCRIPTION
**Pull request by g7morris**

### CODE REVIEW

File: docs/cookbook-recipes/isle-v140-git-cleanup.md [hunk 1]
```diff
@@ -78,7 +78,8 @@ Enter the name here:
 * The script will continue and the process takes about 30 - 45 seconds depending on the speed of your system with minimal output.
 
 * Your final step is to push this local smaller repo back to the remote repository
-* `git push origin master`
+  * `cd ISLE.git` or `cd yourprojectnamehere-isle.git`
+  * `git push origin master`
 
 * Clone down your newly changed and smaller forked or cloned ISLE project directory again to ensure that the repositiory is smaller to a location of your choice.
```
Comment (bseeger) [inline docs/cookbook-recipes/isle-v140-git-cleanup.md]: I think we to say something here to the effect:  *Do not work out of previously cloned git repositories.* You will need to remove any previously existing clones of this repository and clone down new ones.

File: docs/cookbook-recipes/isle-v140-git-cleanup.md [hunk 2]
```diff
@@ -78,7 +78,8 @@ Enter the name here:
 * The script will continue and the process takes about 30 - 45 seconds depending on the speed of your system with minimal output.
 
 * Your final step is to push this local smaller repo back to the remote repository
-* `git push origin master`
+  * `cd ISLE.git` or `cd yourprojectnamehere-isle.git`
+  * `git push origin master`
```
Comment (bseeger) [inline docs/cookbook-recipes/isle-v140-git-cleanup.md]: Per our slack conversation, this needs to be changed to `git push`

File: docs/cookbook-recipes/isle-v140-git-cleanup.md [hunk 3]
```diff
@@ -78,7 +78,8 @@ Enter the name here:
...
 
 * Clone down your newly changed and smaller forked or cloned ISLE project directory again to ensure that the repositiory is smaller to a location of your choice.
```
Comment (bseeger) [inline docs/cookbook-recipes/isle-v140-git-cleanup.md]: Can you make the next line bold?  At least the "Remove all previous versions of your cloned ISLE project directory/directories."  And I think you can take out 'larger size' and 'forked', as I think they will all be clones and 'larger size' and 'previous' are redundant.

Comment (bseeger) [reply docs/cookbook-recipes/isle-v140-git-cleanup.md]: And below that, I think it should read: "Work from a newly cloned ISLE/yourprojecthere-ISLE project directory".

File: docs/cookbook-recipes/isle-v140-git-cleanup.md (hunk 2 above)
Comment (g7morris) [reply docs/cookbook-recipes/isle-v140-git-cleanup.md]: Changed as agreed. Over to you for resolution and approval.

File: docs/cookbook-recipes/isle-v140-git-cleanup.md (hunk 3 above)
Comment (g7morris) [reply docs/cookbook-recipes/isle-v140-git-cleanup.md]: Changed as agreed. Over to you for resolution and approval.

**Pull request merged**